app/javascript/pages/admin/Events/index.js

```diff
@@ -21,16 +21,17 @@ import 'style-loader!css-loader!react-table/react-table.css'
 
 const eventsSort = 'STARTS_AT_DESC'
 
-const actionLinks = (event, deleteEvent) => (
+const actionLinks = (event, deleteEvent, cloneEvent) => (
```

Comment:
`cloneEvent` doesn't appear to be used. I reckon we could remove it.

Reply:
thanks, I forgot to clean this one up! 👍